.ci/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "translators-check",
+  "version": "1.0.0",
+  "description": "Continuous integration of Zotero translators",
+  "main": "index.js",
+  "scripts": {
+    "test": "export SKIP_WARN=true ; find .. -maxdepth 1 -name '*.js' -print0|xargs -0 -n1 -P8 ./checkTranslator.sh 2>/dev/null; ./checkDeletedTxt.sh",
+    "start": "find .. -maxdepth 1 -name '*.js' -print0|xargs -0 -n1 -P1 ./checkTranslator.sh; ./checkDeletedTxt"
```

Comment:
It looks like the `.sh` is missing here at the end. But we could also just call the `checkDeletedTxt.sh` script as a second script in travis. No?